package.json

```diff
@@ -32,29 +32,32 @@
     "tidy:scripts": "eslint --fix test/** scripts/** '*.js' && prettier --write test/** scripts/** '*.js'",
     "size": "npx hardhat size-contracts",
     "coverage": "npx hardhat coverage",
-    "deploy-suite:hardhat": "npx hardhat clean && npx hardhat compile && npx hardhat run --network hardhat scripts/deploy-suite.js",
-    "deploy-suite:local": "npx hardhat clean && npx hardhat compile && npx hardhat run --network localhost scripts/deploy-suite.js",
-    "deploy-suite:test": "npx hardhat clean && npx hardhat compile && npx hardhat run --network test scripts/deploy-suite.js  >> logs/test.deploy.contracts.txt",
-    "deploy-suite:ethereum:mainnet": "npx hardhat clean && npx hardhat compile && npx hardhat run --network mainnet scripts/deploy-suite.js >> logs/mainnet.deploy.contracts.txt",
-    "deploy-suite:polygon:mumbai": "npx hardhat clean && npx hardhat compile && npx hardhat run --network mumbai scripts/deploy-suite.js >> logs/mumbai.deploy.contracts.txt",
-    "deploy-suite:polygon:mainnet": "npx hardhat clean && npx hardhat compile && npx hardhat run --network polygon scripts/deploy-suite.js >> logs/polygon.deploy.contracts.txt",
+    "deploy-suite:hardhat": "npx hardhat clean && npx hardhat compile && npx hardhat deploy-suite --network hardhat",
+    "deploy-suite:local": "npx hardhat clean && npx hardhat compile && npx hardhat deploy-suite --network localhost",
+    "deploy-suite:test": "npx hardhat clean && npx hardhat compile && npx hardhat deploy-suite --network test >> logs/test.deploy.contracts.txt",
+    "deploy-suite:ethereum:mainnet": "npx hardhat clean && npx hardhat compile && npx hardhat deploy-suite --network mainnet >> logs/mainnet.deploy.contracts.txt",
+    "deploy-suite:polygon:mumbai-test": "npx hardhat clean && npx hardhat compile && npx hardhat deploy-suite --network mumbai --env test >> logs/mumbai-test.deploy.contracts.txt",
+    "deploy-suite:polygon:mumbai-staging": "npx hardhat clean && npx hardhat compile && npx hardhat deploy-suite --network mumbai --env staging >> logs/mumbai-staging.deploy.contracts.txt",
+    "deploy-suite:polygon:mainnet": "npx hardhat clean && npx hardhat compile && npx hardhat deploy-suite --network polygon >> logs/polygon.deploy.contracts.txt",
```

Comment:
Don't we need --env prod (or similar) for mainnet deployments? Will it work without "env" being passed in?
And --env test for deployment to the test env.